src/update_manager.cc

```diff
@@ -172,7 +174,7 @@ void UpdateManager::threadProc()
     struct timespec lastUpdate;
     getTimespecNow(&lastUpdate);
 
-    unique_lock<mutex_type> lock(mutex);
+    AutoLockU lock(mutex);
```

Comment:
Have you tested this change? Is it needed? I remember we broke some stuff with a similar change some time ago.

Reply:
i hope i have not changed behavior here. AutoLockU should still resolve to unique_lock. (see using AutoLockU = std::unique_lock<decltype(mutex)>; in update_manager.h)